vllm/forward_context.py

```diff
@@ -82,15 +82,16 @@ def set_forward_context(attn_metadata: Any,
                                          dtype=torch.int32)
         from vllm.distributed.parallel_state import get_dp_group
         dist.all_reduce(num_tokens_tensor, group=get_dp_group().cpu_group)
```

Comment:
since we have the `num_tokens_tensor` , can we keep both `num_tokens_across_dp` and `cum_num_tokens_across_dp` ?

Reply:
we can have a new `dp_metadata` to include both, and I assume we might need more fields in the future.

Reply:
I removed `num_tokens_across_dp` since we weren't using it anywhere but can add it back in. Do you have a future use case for it?

Reply:
it seems deepep will need this: https://github.com/deepseek-ai/DeepEP/blob/c5b4040502f978c866c5f67cb33bd870bba66c06/deep_ep/buffer.py#L261